packages/pds-ember/.template-lintrc.js

```diff
@@ -1,5 +1,9 @@
 'use strict';
 
 module.exports = {
-  extends: 'octane'
+  extends: 'octane',
+
+  rules: {
+    'no-duplicate-landmark-elements': false,
```

Comment:
Why is this one turned off?

Reply:
The one addon code failure is [here](https://github.com/hashicorp/structure/blob/cebe144910ccef7040a19eb64ec49e7366398352/packages/pds-ember/addon/components/pds/global-header/index.hbs#L22) where there are multiple `nav` elements in the file depending on which conditional path is actually triggered (nested blocks strike again).  

There are a handful more in the dummy app so perhaps it'd be best to allow this in existing places but not add a global exception. What do you think?

Reply:
You should be able to give multiple navs aria-labels:

https://github.com/hashicorp/consul/blob/cc5dba1e3b88cc670e75b1fafaf777536962e8c6/ui/packages/consul-ui/app/components/app/index.hbs#L57-L76

> If there is more than one navigation landmark role or <nav> element in a document, provide a label for each landmark.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Navigation_Role#labeling_landmarks

Not sure if the template linter follows those practices tho?

Reply:
Pushed up some changes for this as well

Reply:
Oh great, I guess the linter follows that practice then, excellent! I did notice one more tiny thing there:

```
aria-label="Primary navigation"
```

will read out "Primary navigation navigation" on a screen reader, so you might want to tweak that slightly?

> A brief description of the purpose of the navigation, omitting the term "navigation", as the screen reader will read both the role and the contents of the label.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Navigation_Role#associated_wai-aria_roles_states_and_properties

Reply:
Good catch, thank you!